src/components/ReplyComment/index.js

```diff
@@ -92,7 +95,10 @@ const ReplyCommentId = ({
         </Text>
         <View style={styles.btn} />
       </View>
-      <ScrollView ref={scrollViewRef} contentContainerStyle={styles.commentScrollView}>
+      <ScrollView
+        onContentSizeChange={onSaveHeight}
+        ref={scrollViewRef}
+        contentContainerStyle={styles.commentScrollView}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ScrollView` component now has two new props: `onContentSizeChange` and `ref`. Ensure that these props are being used correctly within the component. The `onContentSizeChange` prop should be a function that handles changes in content size, and the `ref` prop should be a reference to the ScrollView instance.

```diff
-       <ScrollView
+       <ScrollView
+         onContentSizeChange={onSaveHeight}
+         ref={scrollViewRef}
          contentContainerStyle={styles.commentScrollView}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->